README.md

```diff
@@ -1,5 +1,6 @@
 # [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) LORIS Neuroimaging Platform 
 
+<!--alex ignore easy-->
```

Comment:
If we need to start hacking up our documents to add machine annotation for this tool, I don't think it should be run automatically as part of Travis and it adds another dependency (both for developers and Travis). We should just be cognizant of the writing style during code review and someone can periodically run the tool to make sure nothing slipped through.

The link that you provided even says that it's intended to flag things for human review, not be used as an absolute rule.

> As tempting as it is, you don't want to remove every instance that is flagged by alex or your own search. Because, sometimes, these terms can be present out of necessity or as an attempt to be more welcoming.

(That said, the few changes identified in this PR look good to me.)

Reply:
Fair enough, I see your point.

Maybe we could assign someone to run the tool during our release process?

Reply:
Makes sense to me, as long as we remember

Reply:
that's always precisely the issue.   Do we have a new checklist for the release process, that we actually use?

Reply:
The GitHub project functions as a kind of distributed checklist, I guess.

Anyway I've added a `make` target for this tool (not included as a part of Travis though) so that it's quick to run the check anyway. Still a step forward.